hertzbeat-alerter/src/main/java/org/apache/hertzbeat/alert/service/impl/TencentSmsClientImpl.java

```diff
@@ -0,0 +1,194 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hertzbeat.alert.service.impl;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.hertzbeat.alert.config.TencentSmsProperties;
+import org.apache.hertzbeat.alert.service.SmsClient;
+import org.apache.hertzbeat.alert.util.TencentCloudApiSignV3;
+import org.apache.hertzbeat.common.entity.alerter.GroupAlert;
+import org.apache.hertzbeat.common.entity.alerter.NoticeReceiver;
+import org.apache.hertzbeat.common.entity.alerter.NoticeTemplate;
+import org.apache.hertzbeat.common.support.exception.SendMessageException;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.apache.hertzbeat.common.constants.SmsConstants.TENCENT;
+
+/**
+ * sms service client for tencent cloud <br>
+ * doc: <a href="https://cloud.tencent.com/document/api/382/55981">https://cloud.tencent.com/document/api/382/55981</a>
+ */
+
+@Slf4j
+public class TencentSmsClientImpl implements SmsClient {
+
+    private static final String RESPONSE_OK = "Ok";
+    private static final String REGION = "ap-guangzhou";
+    private static final String API_VERSION = "2021-01-11";
+    private static final String ACTION = "SendSms";
+    private static final String HOST = "sms.tencentcloudapi.com";
+    
+    private String appId;
+    private String signName;
+    private String templateId;
+    private String secretId;
+    private String secretKey;
+    private final Gson gson;
+
+    public TencentSmsClientImpl(TencentSmsProperties config) {
+        this.gson = new Gson();
```

Comment:
hi LGTM. Can we use the unified json util `org.apache.hertzbeat.common.util.JsonUtil#fromJson(java.lang.String)` to instead of `Gson`?